parse_comments.py

```diff
@@ -17,12 +26,30 @@
         if "snippet" not in comment_top_level_comment:
             break
         comment_snippet_1 = comment_top_level_comment["snippet"]
+
+        # TEXT_DISPLAY PARSED
         if "textDisplay" not in comment_snippet_1:
             break
         comment_text_display = comment_snippet_1["textDisplay"]
-        comment_list.append(comment_text_display)
-else:
-    ""
+        # LIKE_COUNT PARSED
+        if "likeCount" not in comment_snippet_1:
+            break
+        comment_like_count = comment_snippet_1["likeCount"]
+
+        youtube_comment = YTComment(comment_text_display, comment_like_count, calc_sentiment(comment_text_display))
+        comment_list.append(youtube_comment)
+
+for i in range(1, len(comment_list)):
```

Comment:
So what are you trying to do here?

Reply:
This sorts the sentiment of each comment least to greatest